modules/standard/Regex.chpl

```diff
@@ -1191,17 +1215,22 @@ proc string.search(needle: regex(string), ref captures ...?k):regexMatch
 /* Search the receiving string for a regular expression already compiled
```

Comment:
What is the argument name are you referring to? Here I intended to deprecate `string.search` and `bytes.search` (which return `regexMatch`) and add `string.find` and `bytes.find` (which return `byteIndex`). Since the `search` function isn't exactly the same as the new `find` function, I didn't mention the new `find` function in the deprecation message.

Reply:
Let me rephrase.  You've got two versions of the same method here, the one on line 1210 and the one on line 1228.  The first one says "the needle argument is deprecated", which to me implies that the user can just use the new argument name and avoid the deprecation warning.  However, the pattern version is also now deprecated.  I think it would be frustrating for a user to make one update and then still have a deprecation warning.  We could just say the function name itself is deprecated for both versions, I was just suggesting a way to preserve the distinction between them.

Reply:
Okay, got it.